lib/gcov/README.md

```diff
@@ -0,0 +1,71 @@
+# Unikraft Library for Profiling and Test Coverage
+
+This library introduces initial setup with config options for console, binary file, and memory output. It takes advantage of `GCC's`` Profiling and Test Coverage in Freestanding Environments.
+
+## Prerequisites
+
+This feature was introduced in `GCC-13`, as such, this library won't work with any version lower than that.
+One easy way to build with a specific `GCC` version is to use a `Docker` container - [DockerImage](https://hub.docker.com/_/gcc/)
```

Comment:
It would be worth to add a **Notice:** here that processing should also use with the version of `lcov` that corresponds to the compiler version that supports coverage in freestanding evnironments.